.github/workflows/codeql.yml

```
@@ -0,0 +1,41 @@
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)